ext/gd/gd.c

```diff
@@ -821,6 +821,28 @@ ZEND_BEGIN_ARG_INFO(arginfo_imagesetinterpolation, 0)
 	ZEND_ARG_INFO(0, method)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO(arginfo_imagegifanimbegin, 0)
+	ZEND_ARG_INFO(0, im)
```

Comment:
The first parameter of [ZEND_ARG_INFO()](https://github.com/php/php-src/blob/master/Zend/zend_API.h#L100) indicates whether the argument is passed by reference. Optional arguments would require to use [ZEND_BEGIN_ARG_INFO_EX()](https://github.com/php/php-src/blob/master/Zend/zend_API.h#L115-L117), what I actually should have used. Thanks!